packages/mantine/src/components/table/Table.types.ts

```diff
@@ -76,6 +76,10 @@ export interface TableLayoutProps<T = unknown> {
      * @param datum the row for which the children should be generated.
      */
     getExpandChildren?: (datum: T) => ReactNode;
+    /**
+     * Wether clicking on a selected row will deselect the row or not.
+     */
+    disableRowDeselection?: boolean;
```

Comment:
My small brain has troubles keeping up with the double negative. Can we instead go with something like this?

```suggestion
    keepSelection?: boolean;
```

Reply:
Sure no problem. ✅